test/node/test_front.rb

```diff
@@ -59,4 +59,39 @@ def test_renders_public_pages
       assert(json['all'].find { |r| r['host'] == 'localhost' })
     end
   end
+
+  def test_different_logos
+    {
+      '0' => 'https://www.zold.io/images/logo-red.png',
+      '4' => 'https://www.zold.io/images/logo-orange.png',
+      '16' => 'https://www.zold.io/images/logo-green.png'
+    }.each do |num, path|
+      puts "Calculating score #{num}..."
```

Comment:
@oltdaniel don't use `puts`. Instead, use `$log.info(...)`. This way it will be possible to turn off the entire logging in `test__helper.rb` by replacing `$log` with something else